docusaurus/docs/reference/metadata.md

```diff
@@ -312,13 +323,61 @@ Parameters for the token authentication request.
+| `title`       | string   | **Yes**  | The title of your link extension.                                                                                         |
+| `description` | string   | No       | An optional description of your link extensions.                                                                          |
+
+### exposedComponents
+
+This list has to contain all components that your plugin exposes using `.exposeComponent()`. **Exposed components that are not listed in here won't work.**
+
+#### Properties
+
+| Property      | Type   | Required | Description                                                                                                                                                                                                                 |
+| ------------- | ------ | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`          | string | **Yes**  | An id for your exposed component prefixed with your plugin id. It needs to be unique. A best practice is to also add a version suffix to prevent future breaking changes. E.g.: `myorg-extensions-app/exposed-component/v1` |
+| `title`       | string | No       | The title of your exposed component.                                                                                                                                                                                        |
+| `description` | string | No       | An optional description of your exposed component.                                                                                                                                                                          |
+
+### extensionPoints
+
+This list has to contain all extension points that your plugin defines using `usePluginLinks()` or `usePluginComponents()`. **Extension points that are not listed in here won't work.**
+
+#### Properties
+
+| Property      | Type   | Required | Description                                                                                                                                                                                                             |
+| ------------- | ------ | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`          | string | **Yes**  | An id for your extension point prefixed with your plugin id. It needs to be unique. A best practice is to also add a version suffix to prevent future breaking changes. E.g.: `myorg-extensions-app/extension-point/v1` |
```

Comment:
```suggestion
| `id`          | string | **Yes**  | A unique id for your extension point prefixed with the plugin id. It is recommended to add a version suffix to prevent future breaking changes. E.g.: `myorg-extensions-app/extension-point/v1` |
```